ansible/roles/invoker/tasks/clean.yml

```diff
@@ -9,6 +9,11 @@
   ignore_errors: True
 
 # Unless invoker cleans up its own containers we have to do this here.
+#  shell: "(docker ps -aq --no-trunc --filter name=wsk | sudo xargs docker-runc resume); sleep 5"
+- name: pause at runc-level to restore docker consistency
```

Comment:
I think now is the time to make that change in the invoker. I recall having opened an issue "invoker should clean up on terminate" or something way back in time. We agreed to dismiss it at the time and have ansible clean up manually. 

@perryibm how much effort would it be to unpause and remove all action containers when the invoker receives a SIGTERM?